kernel/README.md

```diff
@@ -16,23 +16,99 @@ sets up a kernel. If you want to add a source modification, add a patch on this
 directory.
 
 The script also adds a kernel config file from
-`${GOPATH}/src/github.com/kata-containers/packaging/kernel/configs/` to .config
+`${GOPATH}/src/github.com/kata-containers/packaging/kernel/configs/` to `.config`
 in the kernel source code. You can modify it as needed.
 
 # Build the kernel
 
-After the kernel source code is ready it is possible to build the kernel.
+After the kernel source code is ready, it is possible to build the kernel.
 
 ```bash
-./build-kernel.sh build
+$ ./build-kernel.sh build
 ```
 
 
 ## Install the Kernel in the default path for Kata
 
 Kata Containers uses some default path to search a kernel to boot. To install
 on this path, the following command will install it to the default Kata
-containers path.
+containers path (`/usr/share/kata-containers/`).
+
+```bash
+$ ./build-kernel.sh install
+```
+
+## Submit Kernel Changes
+
+Kata Containers packaging repository holds the kernel configs and patches. The
+config and patches could be for many versions, but we only make sure to test the
```

Comment:
Suggested rewrite:
 config and patches can work for many versions, but we only test the